Python/Util/pyDDSurface.cpp

```diff
@@ -77,10 +77,19 @@ PY_METHOD_VA(DDSurface, setFromMipmap,
     Py_RETURN_NONE;
 }
 
-PY_METHOD_NOARGS(DDSurface, createMipmap,
+PY_METHOD_VA(DDSurface, createMipmap,
+    "Params: Mipmap name"
     "Create a new plMipmap from this plDDSurface")
 {
-    return ICreate(self->fThis->createMipmap());
+    const char* name;
+    if (!PyArg_ParseTuple(args, "s", &name)) {
```

Comment:
I feel like this should be optional, in order to a) not break existing code, and b) more closely match the general `hsKeyedObject` constructor.  The key should be `init`ed either way, but with an empty string if no parameter is passed (again, to match `hsKeyedObject`).

Reply:
I didn't think that a blank key name would be valid (it causes silent failures when I did that during testing, as only the first one is kept).  That's still a better solution than no KO at all, as you can easily set the key name from Python.